tests/rindle.spec.js

```diff
@@ -33,499 +34,526 @@ var EventEmitter = require('events').EventEmitter;
 var tmp = require('tmp');
 tmp.setGracefulCleanup();
 
-var rindle = require('../lib/rindle');
+chai.use(require('chai-as-promised'));
+chai.use(require('sinon-chai'));
 
-describe('Rindle:', function() {
-  'use strict';
+var { expect } = chai;
```

Comment:
This was what was necessary for linting, however I also removed mochainon in the process of working that out and decided it made sense to keep it removed